flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "automerge-flathubbot-prs": true,
```

Comment:
Don't automerge PRs, they'll get merged without any oversight

Reply:
That's what has already been done for the previous php extensions.

As the maintainer, I'm OK with it.

Reply:
Okay, I've removed it.